index.html

```diff
@@ -0,0 +1,83 @@
+
+    <title>Tracking The Blue</title>
+
+  </head>
+  <body>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+
+    <div id =content class="d-flex justify-content-center" >
+ <!--TODO: need to fix Jumbotron so that it fills the screen, jmbotron-fluid should
+        Have fixed this, but it fills it's parrent element, but it's parrent item doese not fill the screen-->
+          
+    <div class="container-fluid">
+        <div class="jumbotron jumbotron-fluid" style="background-color: blue; color: yellow;">
+        <div>
+            <h1>Tracking the Blue</h1>
+        </div>
+
+        </div>
+    
+        <div id="about_the_project">
+            <p>Isum Copy... sdfskdhf;kjhsdjfh</p>    
+                <p>sdfjhk sdjfhksdhf</p> 
+                <p>skdkfjlsdkjf sdfkjsdl sdlfkj dirn sldkfj</p> 
+                <p>KDSjflskjd  twefkljs kjliji nfdsnl. </p>
+            </p>
+        </div>
+        <div >
+            <div id="citation_information">
+                <p>Citation Information</p>
+            </div>
+        </div>
+        
+        <form  action="/action_page.php">
+        
+            <div class="form-group">
+                <label for="citation_number" >Citation #</label>
+                <input type ="text" class="form-control" id="citation_number">
+            </div>
+        
+            <div class="form-group">
+                <label for="date_issued" >Date citation was issued</label>
+                <input type ="datetime-local" class="form-control" id="date_issued">
+            </div>
+
+            <div class="form-group">
+                <label for="City" >City</label>
+                <input type ="text" class="form-control" id="city">
+            </div>
+
+            <div class="form-group">
+                <label for="State" >State</label>
+                <input type ="text" class="form-control" id="State">
+            </div>
+
+            <div class="form-group">
+                <label for="Prestinct" >Prestinct</label>
+                <input type ="text" class="form-control" id="Prestinct">
+            </div>
+
+        </form>
+        
+        <div class="form-group d-flex justify-content-center">
+                
+            <a href="C:\Users\User\OneDrive\Desktop\Tracking the Blue\Civi_into.html"><button type="submit" class="btn btn-default">Your Info</button></a>

```

Comment:
we're going to want the change the href to be a [relative path](https://stackoverflow.com/questions/24028561/relative-path-in-html#24028813) to the directory that we're in.